Add Pump Support Coverage to LTS page (#2183)
TykTechnologies/tyk-docs: tyk-docs/content/frequently-asked-questions/long-term-support-releases.md

| R6      | February 2024 | May 2024                    | May 2025              | May 2026                  |
| R7      | February 2025 | May 2025                    | May 2026              | May 2027                  |

## Pump

As updates to Tyk Pump are deployable quickly with low risk and no breaking changes, Tyk will support the latest major version (I.e Pump 1.7) until the next major version (1.8) is released. Then we'd support that version.

To help assure backward compatibility we ensure that the each version of Pump we release works with the gateway and dashboard versions which is under long term support at that time. For example, we'd ensure Pump 1.7 is compatible with release 4 of our gateway and dashboard, and this increments with the long term support model.

If we need to patch a major Pump version , we would number that as a minor version (1.7.1).

## Other Components

> Our Tyk Identity Broker (TIB),  MDCB, and Operator components will not be a part of our long term support policy yet. We will continue to support all versions of these components, and will offer advice to clients on when to upgrade if necessary.

Reviewer: ```suggestion
Our Tyk Identity Broker (TIB),  MDCB, and Operator components are not part of our Long-Term Support policy yet. We will continue to support all versions of these components and will offer advice to clients on when to upgrade if necessary.
```

## Other Components

> Our Tyk Identity Broker (TIB),  MDCB, and Operator components are not part of our Long-Term Support policy yet. We will continue to support all versions of these components and will offer advice to clients on when to upgrade if necessary.

Reviewer: ```suggestion
Our Tyk Identity Broker (TIB),  MDCB, and Operator components are not part of our Long-Term Support (LTS) policy yet. We will continue to support all versions of these components and will offer advice to clients on when to upgrade if necessary.
```
